src/engine/state/phase.ts

```diff
@@ -30,6 +30,9 @@ export enum Phase {
 
   // Montreal Metro
   GOVERNMENT_BUILD = 16,
+
+  // Poland
+  POLAND_GOODS_GROWTH = 17,
```

Comment:
Instead of creating a new phase, can you just add the action to the existing phase?